### PR TITLE
chore: bump configuration to v0.1.0-rc.15

### PR DESCRIPTION
**Pull request by Imti**
## Motivation

Need to update packages with new configuration package version

## Solution

Update packages with new configuration package version

## PR Checklist

- [ ] Added Tests (NA)
- [ ] Updated Documentation (NA)
- [x] Updated CHANGELOG.md for the appropriate package

### CODE REVIEW

File: packages/new-deploy/changelog.md [hunk 1]
```diff
@@ -13,3 +13,4 @@
 - store governance CallBatch on DeployContext instead of returning
 - update typescript eslint packages to fix lint script
 - standardize prettier
+- chore: bump configuration to v0.1.0-rc.14
```
Comment (ErinHales) [inline packages/new-deploy/changelog.md:16]: It's rc.15

File: packages/sdk-bridge/changelog.md [hunk 2]
```diff
@@ -2,9 +2,12 @@
 
 ### Unreleased
 
+### 1.0.0-rc.9
+
 - update typescript eslint packages to fix lint script
 - remove unnecessary lint:fix script
 - standardize prettier
+- chore: bump configuration to v0.1.0-rc.14
```
Comment (ErinHales) [inline packages/sdk-bridge/changelog.md:10]: 15

**Review by ErinHales — APPROVED** [review]
All the change logs say 14, other than that looks good

**Comment by Imti** [comment]
> All the change logs say 14, other than that looks good

Ah got my numbers mixed up.

**Pull request merged**